### ISSUE TITLE
Simple suggestion

### ISSUE THREAD

**Issue by anilanar** [Pauan/rollup-plugin-purs#17]
As far as I can see, this plugin requires that the entry file must have a main function exported, otherwise it doesn't have much to do with purescript. If you remove that restriction (remove `import {main} from './entry'; main()`), perhaps this plugin can be simply renamed to`rollup-plugin-uncurry`?

**Comment by Pauan** [comment]
Yes, that's true, 99% of the functionality of this plugin works with JavaScript in general, it's not specific to PureScript.

I've thought about splitting it into separate plugins, but that increases the amount of configuration that the user has to do.

**Comment by anilanar** [comment]
Maybe can split this plugin into multiple, and import them in this plugin without even breaking backward compat.?

Otherwise, if pureVars and runMain were false by default, nobody would mind some purescript-specific configs so not splitting is also fine.
